analysis_template/plhld/ml/test.py

```diff
@@ -0,0 +1,97 @@
+# coding: utf-8
+
+"""
+Test model definition.
```

Comment:
we can remove this test ml model since it is already included in columnflow

Reply:
Removed.